### PR TITLE
Feature/iia 934 add stamp to pattern create

### PR DESCRIPTION
**Pull request by dholubek**
This adds the STAMP menu
updated the title panes in the menu to default to blank except the status is set to active,
this does touch concept but I tested that and its works
it also updates the "banner" at the top of the pattern to grab the FQN, publicId, and real stamp time instead of an in memory time placeholder

image

image

### CODE REVIEW

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java [hunk 1]
```diff
@@ -216,7 +221,26 @@ public boolean createPattern() {
...
+
+    public String getPatternTitle() {
+        return ((EntityFacade) getPropertyValue(PATTERN)).description();
+    }
+
+    public Long getPatternLastStampTime() {
+        EntityFacade patternFacade = getPropertyValue(PATTERN);
+        Entity pattern = EntityService.get().getEntityFast(patternFacade.nid());
+        IntIdSet idSet = pattern.stampNids();
+        OptionalLong latestStamp = idSet.intStream().mapToLong(i -> ((StampEntity)EntityService.get().getEntityFast(i)).time()).max();
```
Comment (carldea) [inline kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java:238]: Can we use `getViewProperties().stampCalculator()`?

Comment (dholubek) [reply kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java]: updated to use the StampCalculator along with the code that @rbradley813 suggested to get the STAMP.

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java [hunk 2]
```diff
@@ -216,7 +219,36 @@ public boolean createPattern() {
...
+        stampViewModel.setValue(TIME, stamp.time());
+        stampViewModel.setValue(AUTHOR, stamp.author());
+        stampViewModel.setValue(MODULE, stamp.module());
+        stampViewModel.setValue(PATH, stamp.path());
+    }
 
-        return composer.commitSession(session);
+    public String getPatternIdentifierText() {
+        EntityFacade patternFacade = getPropertyValue(PATTERN);
+        return String.valueOf(patternFacade.toProxy().publicId().asUuidList().getLastOptional().get());
```
Comment (carldea) [inline kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java:252]: Can we look at how our recent change from the concept window? 
It has code to also display the SNOMED and LOINC codes if present.

**Review by carldea — APPROVED** [review]
Looks good. 
I do have an optional suggestion / question. But if you want to defer to another ticket let me know. Since I will approve, go ahead an merge either way you decide above.

**Pull request merged**